corehq/apps/builds/management/commands/commcare_build_importer.py

```diff
@@ -101,8 +101,8 @@ def add_build(self, builf_file, version, build_number):
     def _extract_version_numbers(self, build_dict, max_options=5):
         """
         - takes build_dict is of following format
-            {352167: 'http://jenkins.dimagi.com/job/commcare-mobile-2.21/352167/',
-             352366: 'http://jenkins.dimagi.com/job/commcare-mobile-2.21/352366/'}
+            {352167: 'http://jenkins.dimagi.com/job/commcare-mobile-2.21.0/352167/',
+             352366: 'http://jenkins.dimagi.com/job/commcare-mobile-2.21.0/352366/'}
```

Comment:
Did you look at Jenkins? If you had you may have noticed that none of the jobs include the `point` in the name. The reason being that the same build job will be used for all point releases of that version.

Reply:
Didn't refer Jenkins. Was just looking at how we mention versions on HQ. Would this change effect things on jenkins?

Reply:
to be reverted.